.github/actions/frontend/runtime/validate/action.yml

```diff
@@ -6,9 +6,20 @@ name: Runs validation on the repo
 runs:
   using: "composite"
   steps:
+    - name: check_for_auth_token
```

Comment:
@Sivli-Embir Added per this convo https://jptrone.slack.com/archives/D03L33K6HE3/p1692214922442019